projects/storefrontlib/src/cms-components/myaccount/payment-methods/payment-methods.component.html

```diff
@@ -1,38 +1,40 @@
-<div class="cx-payment container">
-  <div class="cx-header">
-    <h3>{{ 'paymentMethods.paymentMethods' | cxTranslate }}</h3>
-  </div>
-
-  <div class="cx-body">
-    <div class="cx-msg">
-      {{
-        'paymentMethods.newPaymentMethodsAreAddedDuringCheckout' | cxTranslate
-      }}
+<ng-container *ngIf="paymentMethods$ | async as paymentMethods">
+  <div class="cx-payment container">
+    <div class="cx-header">
+      <h3>{{ 'paymentMethods.paymentMethods' | cxTranslate }}</h3>
     </div>
-    <div *ngIf="loading$ | async; else cards"><cx-spinner></cx-spinner></div>
-    <ng-template #cards>
-      <div
-        *ngIf="paymentMethods$ | async as paymentMethods"
-        class="cx-existing row"
-      >
+
+    <div class="cx-body">
+      <div class="cx-msg">
+        {{
+          'paymentMethods.newPaymentMethodsAreAddedDuringCheckout' | cxTranslate
+        }}
+      </div>
+      <div *ngIf="loading$ | async; else cards"><cx-spinner></cx-spinner></div>
+      <ng-template #cards>
         <div
-          class="cx-payment-card col-sm-12 col-md-12 col-lg-6"
-          *ngFor="let paymentMethod of paymentMethods"
+          *ngIf="paymentMethods$ | async as paymentMethods"
```

Comment:
Since we now have this logic at the beginning of the the template, we should probably remove `*ngIf="paymentMethods$ | async as paymentMethods"` altogether on line 16.